src/pages/resources/components/state/prices.svelte

```diff
@@ -156,9 +166,7 @@
                 <div class="offer">
                     <div class="service">Staking</div>
                     <div class="price">
-                        {(Number($stakingPrice.value) * 1000).toFixed(
-                            $stakingPrice.symbol.precision
-                        )}
+                        {$stakingPrice.value.toFixed($stakingPrice.symbol.precision)}
```

Comment:
I'm confused by this 😅 We were off by a factor of 1000??

Reply:
Ahh it's because we added the multiplicaiton by 1000 [here](https://github.com/greymass/unicove/pull/211/files#diff-df9cf161ab2cb650e82e72adc950f1945a437e89fe5ce8b728a9b32c420af4beR137) 😄